program/src/state.rs

```diff
@@ -200,7 +200,9 @@ pub struct MangoGroup {
     pub msrm_vault: Pubkey,
     pub fees_vault: Pubkey,
 
-    pub padding: [u8; 32], // padding used for future expansions
+    pub max_mango_accounts: u32, // limits maximum number of MangoAccounts.v1 (closeable) accounts
```

Comment:
could you elaborate on how this would actually be used? code looks ok, but I am lost why we need this